README.md

```diff
@@ -49,7 +49,21 @@ var constructorSessionId = ConstructorIo.getSessionId()
 
 In most cases, you will want to store those parameters as cookies preferably as **ConstructorioID_client_id** and **ConstructorioID_session_id** to be sent with your requests to your backend servers.
 
-## 5. Request Autocomplete Results
+## 5. Setting test cell information for A/B tests
+When A/B testing, it is important to specify which cell the user is being assigned to. Information about the test cell can be set through the `ConstructorIoConfig` object.
+
+```kotlin
+var ConstructorIoConfig(
+  apiKey = "YOUR API KEY",
+  serviceUrl = "ac.cnstrc.com", // default
+  testCells = listOf("constructorio_test" to "control_1")
+)
+
+// Create the client instance
+ConstructorIo.init(this, config)
```

Comment:
Same thing as for iOS.

Can we add the test cells editing snippet.


ConstructorIo.testCells = listOf("Constructorio_Test" to "Control_1/Control_2/Constructorio_1/Constructorio_2")

Reply:
Yessirrrr. Forgot about that. Will add and ship. Thanks!